.github/workflows/selftest.yml

```diff
@@ -74,4 +74,17 @@ jobs:
         - name: Convert CITATION.cff from a subdirectory to Zenodo metadata format, show result on stdout
           uses: ./
           with:
-            args: "--infile ./tests/subdirectory/CITATION.cff -f zenodo"
+            args: "--infile ./tests/subdirectory/CITATION.cff --format zenodo"
+
+  convert-to-zenodo-and-compare:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Clone this repository
+          uses: actions/checkout@v2
+        - name: Convert CITATION.cff from a subdirectory to Zenodo metadata format, show result on stdout
+          uses: ./
+          with:
+            args: "--infile ./tests/zenodo-equivalent/CITATION.cff --format zenodo --outfile .zenodo.test.json"
```

Comment:
I would prefer to put the generated file in ./tests/zenod-equivalent/  if that works. The diff below should then also be changed.